include/riak_cs_gc.hrl

```diff
@@ -106,3 +93,14 @@
 -define(DEFAULT_GC_BATCH_SIZE, 1000).
 -define(DEFAULT_GC_WORKERS, 2).
 -define(EPOCH_START, <<"0">>).
+
+-record(gc_manager_state, {
+          next :: undefined | non_neg_integer(),
+          gc_batch_pid :: undefined | pid(),
+          batch_history = [] :: list(#gc_batch_state{}),
```

Comment:
Well, thanks for agreement. Although it's good idea I think, somewhat it's out of the scope of refactoring and will do in future. For now, the history will be a list of just one element. Please see the latest code in manager receiving `finished` message.